docs/related_packages/pecos.rst

```diff
@@ -0,0 +1,68 @@
+.. _pecos:
+
+Pecos: Data Quality Control and Fault Detection
+=================================================
+
+Before using plant data in process models, quality control and fault detection analysis is recommended to identify 
+potential data issues (i.e., missing or corrupt data) and data points that are not suitable for the intended analysis 
+(i.e., abnormal plant behavior).
+This following documentation describes methods to run data quality control and fault detection analysis using Pecos.
```

Comment:
The following...

or 

This documentation...

Reply:
Done